html/about.html

```diff
@@ -35,7 +40,7 @@ <h5 class="creative-logo-title">Creative Design & Content 2018 Logo</h5>
       <p class="creative-description">The logo of Design was decided through the logo competition</p>
       <a class="logo" href="../index.html">Design & Content</a>
     </section>
-    <section class="past-events">
+    <section class="past-events" id="events">
       <h5 class="past-title">See the past Design & Content Conferences </h5>
       <hr class="orange-line">
       <p class="past-description">Take a look at the last two Design & Content Conferences</p>
```

Comment:
- [ ] This part of the footer should not be part of the about page for desktop version:
![Screenshot from 2021-11-16 18-08-14](https://user-images.githubusercontent.com/62705947/142032107-5f5723d7-a81a-4a20-ba9d-cd230c8fbd92.png)

Reply:
i 've just changed it thanks again